js/Common.js

```diff
@@ -1337,13 +1337,14 @@ CRM.strings = CRM.strings || {};
    * @return string
    */
   var currencyTemplate;
-  CRM.formatMoney = function(value, format) {
+  CRM.formatMoney = function(value, format, onlyNumber) {
     var decimal, separator, sign, i, j, result;
     if (value === 'init' && format) {
       currencyTemplate = format;
       return;
     }
     format = format || currencyTemplate;
+    onlyNumber = onlyNumber || false;
```

Comment:
Yes, but since it is a boolean, and defaults to false, and undefined == false in javascript, the extra line is unnecessary.